articles/azure/azs-how-create-vm-ps.md

```diff
@@ -0,0 +1,288 @@
+---
+title: How to create a Virtual Machine - PowerShell | UKCloud Ltd
+description: Provides help for creating a virtual machine on UKCloud for Microsoft Azure using PowerShell
+services: azure-stack
+author: Bailey Lawson
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Create a Virtual Machine
+toc_sub3:
+toc_sub4:
+toc_title: Create a Virtual Machine - PowerShell
+toc_fullpath: Users/How To/Create a Virtual Machine/azs-how-create-vm-ps.md
+toc_mdlink: azs-how-create-vm-ps.md
+---
+
+# How to create a Virtual Machine in Azure Stack with PowerShell

```

Comment:
Change heading to:
How to create a virtual machine using PowerShell

Reply:
Fixed